spring-cloud-stream-app-descriptor/src/main/resources/META-INF/kafka-apps-docker.properties

```diff
@@ -43,6 +45,9 @@ sink.rabbit=docker:springcloudstream/rabbit-sink-kafka:@docker.tag@
 sink.redis-pubsub=docker:springcloudstream/redis-sink-kafka:@docker.tag@
 sink.router=docker:springcloudstream/router-sink-kafka:@docker.tag@
 sink.s3=docker:springcloudstream/s3-sink-kafka:@docker.tag@
+sink.sftp=docker:springcloudstream/sftp-sink-kafka:@docker.tag@
+sink.task-launcher-local=docker:springcloudstream/task-launcher-local-sink-kafka:@docker.tag@
+sink.task-launcher-yarn=docker:springcloudstream/task-launcher-yarn-sink-kafka:@docker.tag@
```

Comment:
`integration-test` is only used by the automated tests, there is no reason to register it in SCDF